code/game/gamemodes/dynamic/dynamic_rulesets_midround.dm

```diff
@@ -579,7 +570,12 @@
 	)
 	required_enemies = list(2,2,1,1,1,1,1,0,0,0)
 	required_candidates = 1
+<<<<<<< HEAD
 	weight = 0 //SKYRAT EDIT CHANGE
```

Comment:
You guys really should just use dynamic.json config instead of doing this

Reply:
We should

Reply:
Yeah, we should have everything go by dynamic.json instead of hardcoding it in. That's how it should be.